src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -296,16 +319,16 @@ export default class PhotoField extends React.Component {
           {!this.state.done && this.state.src && <div className="cropper-container-outer">
             <Cropper
               ref="cropper"
+              ready={this.setCropBox}
               responsive
               src={this.state.src}
-              aspectRatio={1 / 1}
+              aspectRatio={1}
               cropBoxMovable={false}
               toggleDragModeOnDblclick={false}
               dragMode="move"
               guides={false}
               viewMode={1}
-              zoom={this.onZoom}
-              crop={this.cropImage}/>
```

Comment:
Good idea removing this since we never crop by manually setting the crop box or by using autocrop, etc.